sunpy/timeseries/sources/eve.py

```diff
@@ -18,7 +18,7 @@
 from sunpy.util.metadata import MetaDict
 from sunpy.visualization import peek_show
 
-__all__ = ['EVESpWxTimeSeries', 'ESPTimeSeries']
+__all__ = ["EVESpWxTimeSeries", "ESPTimeSeries"]
```

Comment:
Did you run a linter?

Reply:
No I ran codestyle on it, isn't this what we should be using instead?

Reply:
What do you mean? The code style shouldn't make any of these changes.

Reply:
Oh yeah, my apologies. These were changed by `autopep8`.